frontend/lib/src/components/core/LibContext.tsx

```diff
@@ -82,10 +82,17 @@ export interface LibContextProps {
    */
   currentPageScriptHash: string
 
-  /** The lib-specific configuration from the apps host which is requested via the
+  /**
+   * The lib-specific configuration from the apps host which is requested via the
    * _stcore/host-config endpoint.
    */
   libConfig: LibConfig
+
+  /**
+   * If the current script run corresponds to a fragment, the ID of that
```

Comment:
super-nit: `... to a fragment, its the ID of that fragment.`

Reply:
Hm, I feel like that ends up being slightly awkward-sounding too. Reworded this docstring a bit